src/mongoose_lib.erl

```diff
@@ -17,6 +17,8 @@
 -ignore_xref([pairs_foreach/2, wait_until/3]).
 
 -export_type([microseconds/0]).
+-export([pmap/2, pmap/3]).
+-ignore_xref([pmap/3]).
```

Comment:
If `pmap/2` calls `pmap/3`, and `pmap/2` doesn't need the ignore_xref, `pmap/3` shouldn't need it either.

Reply:
Sure, xref disagrees:

```
src/mongoose_lib.erl:190: Warning: mongoose_lib:pmap/3 is unused export (Xref)
```

Reply:
Really? Ah stupid xref 🤷🏽‍♂️

Reply:
not stupid. pmap/3 is only called from tests :D 
And pmap/2 is called from mongoose_async_pools.
But pmap/2 calls pmap/3 using a local call.